relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -152,13 +158,26 @@ pub fn extract_session_metrics<T: SessionLike>(
         ));
 
         if let Some(distinct_id) = nil_to_none(session.distinct_id()) {
+            let mut tags_for_abnormal_session = tags.clone();
```

Comment:
Out of curiosity, why do we only need this tag on the `user` metric, and not on the session count?